src/containers/my-resources/add-resource-with-input/AddResourceWithInput.tsx

```diff
@@ -41,7 +41,7 @@ const AddResourceWithInput: FC<AddResourceWithInputProps> = ({
   button,
   selectedItems,
   setItems,
-  hideCategoriesFilter
+  showNoneProperty
```

Comment:
remove this prop, you don`t need to pass it from each resources container

Reply:
done